ui/src/app/services/api/mock-app-fixures.ts

```diff
@@ -57,23 +61,23 @@ export const bitcoinI: AppInstalledFull = {
   instructions: 'some instructions',
   lastBackup: new Date().toISOString(),
   configuredRequirements: [],
-  hasFetchedFull: true,
-  ui: false,
+  lanUi: false,
+  torUi: false,
```

Comment:
as far as I can tell the API has never returned ui available on AppInstalledFull. We can amend it to but that isn't how it works right now.